target/linux/mpc85xx/p1020/config-default

```diff
@@ -43,6 +43,7 @@ CONFIG_RPS=y
 CONFIG_RWSEM_SPIN_ON_OWNER=y
 CONFIG_SMP=y
 CONFIG_SPI_GPIO=y
+CONFIG_SWCONFIG=y
```

Comment:
@BKPepe any idea why swconfig is still needed?

Reply:
https://github.com/openwrt/openwrt/pull/10850#issuecomment-1279501408